src/utils/CrossChainBridge.ts

```diff
@@ -124,10 +135,17 @@ export const createAndReturnSideToken = (sideTokenAddress: Address, event: NewSi
 
 export const handleFederatorVoted = (event: VotedEvent, transaction: Transaction): void => {
```

Comment:
This function needs to be renamed or moved to a mapping file

Reply:
why?

Reply:
CrossChainBridge is the util file meant to deal with everything bridge related this is why this is here
it is handling all voted events (there are 3 handlers for voted events since the event signature had changed over time)

Reply:
Right, but in the code guidelines we decided that functions named handle... should be in a mapping file. If it's just a way to handle the different event signatures, it should be in the mapping file. If it's dealing with modifying a specific entity, it should be in utils

Reply:
I changed the method name